swarm/pss/protocol.go

```diff
@@ -233,6 +243,21 @@ func (self *Protocol) AddPeer(p *p2p.Peer, topic Topic, asymmetric bool, key str
 	return rw, nil
 }
 
+func (self *Protocol) RemovePeer(asymmetric bool, key string) {
+	log.Debug("closing pss peer", "asym", asymmetric, "key", key)
+	self.RWPoolMu.Lock()
+	defer self.RWPoolMu.Unlock()
+	if asymmetric {
```

Comment:
If asymmetric is of the wrong value, this function will panic. I am suggesting to ether check if pubKeyPool or symKeyPool rw should be closed, or to remove the asymmetric argument and self discover which rw to close.